ui/helpers/job.js

```diff
@@ -110,6 +110,7 @@ export const findSelectedInstance = function findSelectedInstance() {
   if (selectedEl) {
     return findInstance(selectedEl);
   }
+  return null;
```

Comment:
I wasn't sure what to return at many of these places after enabling ```'consistent-return': 'off'```, so I just returned null. Is this the correct way to go about this?

Reply:
For all of the `consistent-return` changes, add `//eslint-disable-next-line consistent-return` at the top of the function (e.g. before `findSelectedInstance`) instead of adding `return null;` at the end. This will guarantee that you're clearly not changing the behaviour of the function and the code coverage score will not be affected (since you're only adding a comment). Adding a new line that is not tested will decrease the total number of lines not tested. Minimizing code coverage changes will help you more clearly see if something is regressing.

Right now, this PR makes the code coverage score globally drop 0.08 while it reduces the coverage a ~35% relatively:
<img width="260" alt="image" src="https://user-images.githubusercontent.com/44410/67031610-e3b25800-f0df-11e9-8cfa-0629f8b3e323.png">

Now, we might actually not be functionally regressing anything but extra code coverage output makes it harder to notice.